cookbook/agent_concepts/knowledge/vector_db/lance_db/lance_db.py

```diff
@@ -0,0 +1,23 @@
+from agno.agent import Agent
+from agno.knowledge.knowledge import Knowledge
+from agno.vectordb.lancedb import LanceDb
+
+# Create Knowledge Instance with LanceDB
+knowledge = Knowledge(
+    name="Basic SDK Knowledge Base",
+    description="Agno 2.0 Knowledge Implementation with LanceDB",
+    vector_store=LanceDb(
+        table_name="vectors",
+        uri="tmp/lancedb",
+    ),
+)
+
+knowledge.add_content(
+    name="Recipes",
+    url="https://agno-public.s3.amazonaws.com/recipes/ThaiRecipes.pdf",
+    metadata={"user_tag": "Recipes from website"},
```

Comment:
```suggestion
    metadata={"doc_type": "recipe_book"},
```

Just made it up, but feels more relevant